workflow-service/src/main/java/com/redhat/parodos/workflow/execution/entity/WorkFlowExecution.java

```diff
@@ -70,6 +70,13 @@ public class WorkFlowExecution extends AbstractEntity {
 	@Column(name = "project_id")
 	private UUID projectId;
 
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "original_workflow_execution_id")
+	private WorkFlowExecution originalWorkFlowExecution;
+
+	@OneToMany(mappedBy = "originalWorkFlowExecution", cascade = { CascadeType.ALL }, fetch = FetchType.LAZY)
+	private List<WorkFlowExecution> restartedWorkFlowExecutions = new ArrayList<>();
```

Comment:
Why this is needed? or is it relevant only when originalWorkFlowExecution is null?

Reply:
I use this to get the amount of time a workflow is restarted

I am still unsure if I want to use that or just use a count request because if there are hundred of restarts, it may take a toll on the memory

Reply:
So I removed this and I am executing count request